### PR TITLE
chore: Replace `if` statement with `if` expression ✨

### PR DESCRIPTION
**Pull request by yezz123**

### CODE REVIEW

**Comment by barbieri** [comment]
I'm not sure what's this PR about, could you elaborate?

Does it fix any bugs or it's purely cosmetic? I really appreciate contributions, but I'd like to avoid noise with cosmetic changes (even more some that I personally dislike :-D like the use of ternary `value if condition else other`)

File: sgqlc/codegen/schema.py [hunk 1]
```diff
@@ -92,11 +92,7 @@ def to_docstring(wrapped_text, level=1):
     if not wrapped_text:
         return ''
     prefix = '    ' * level
-    if len(wrapped_text) == 1:
-        suffix = ''
-    else:
-        suffix = '\n' + prefix
-
+    suffix = '' if len(wrapped_text) == 1 else '\n' + prefix
```
Comment (barbieri) [inline sgqlc/codegen/schema.py:95]: this one makes it shorter for sure, but I really dislike these ternary in Python as it reminds me of Perl.

File: sgqlc/codegen/schema.py [hunk 2]
```diff
@@ -181,10 +177,7 @@ def write_banner(self, text):
 
     @staticmethod
     def has_iface(ifaces, name):
-        for iface in ifaces:
-            if name == iface['name']:
-                return True
-        return False
+        return any(name == iface['name'] for iface in ifaces)
```
Comment (barbieri) [inline sgqlc/codegen/schema.py:180]: this one I'm okay, but given how small is the method, I'd not bother with such change

File: sgqlc/codegen/schema.py [hunk 3]
```diff
@@ -195,7 +188,7 @@ def depend_cmp(a, b):
             return -1
         elif a_ifaces and not b_ifaces:
             return 1
-        elif not a_ifaces and not b_ifaces:
+        elif not a_ifaces:
```
Comment (barbieri) [inline sgqlc/codegen/schema.py:191]: indeed redundant, thanks

File: sgqlc/codegen/schema.py (hunk 2 above)
Comment (yezz123) [reply sgqlc/codegen/schema.py:180]: A common pattern is that we need to find if some condition holds `for` one or all of the items in a collection.

Using Python's `any()` built-in functions is a more concise way of doing this than using a `for` loop.

`any()` will return True when at least one of the elements evaluates to True.

File: sgqlc/codegen/schema.py (hunk 3 above)
Comment (yezz123) [reply sgqlc/codegen/schema.py:191]: Yes, This one looks better 🚀✨

**Comment by yezz123** [comment]
> I'm not sure what's this PR about, could you elaborate?
> 
> Does it fix any bugs or it's purely cosmetic? I really appreciate contributions, but I'd like to avoid noise with cosmetic changes (even more some that I personally dislike :-D like the use of ternary `value if condition else other`)

Hey @barbieri My PR was just refactoring some functions and improving the code to be shorter and fast and respecting all PEP's rules 🚀

File: sgqlc/codegen/schema.py (hunk 2 above)
Comment (barbieri) [reply sgqlc/codegen/schema.py:180]: yeah, I know how `any()`  works :-)

**Comment by barbieri** [comment]
I tried to fix the github action checks, please rebase and push-force it so they will run again?

**Comment by barbieri** [comment]
sorry taking so long, I was really busy these weeks... I did the rebase and pushed as 465a5e800f8b408ceafe25cde45ee0bde4912482. Thanks for contributing!

**Pull request closed without merging**